docs/book/intro.md

```diff
@@ -8,7 +8,7 @@ if a user correctly provides the first word, the second is likely correctly ente
 improve OCR software for digitizing books.
 
 <!-- markdownlint-disable-next-line header-increment -->
```

Comment:
```suggestion
<!-- markdownlint-disable-next-line heading-increment -->
```

The name of this issue was changed in markdown lint

Reply:
Its done.